### PR TITLE
Fix VOput() bug reported by grahameblackwood

### PR DESCRIPTION
**Pull request by dahu**

### CODE REVIEW

File: vimoutliner/plugin/votl_smart_paste.vim [hunk 1]
```diff
@@ -4,10 +4,10 @@
 " middle of a branch.
 " Added 2011-03-01(JB): This script will now also copy an outline correctly by
 " using \\y, copy to the register with \\r, cut an outline by using \\d, and
-" paste from the register using \\b. 
+" paste from the register using \\b.
 " http://www.lists.vimoutliner.org/pipermail/vimoutliner/2008-October/002366.html
 
-map <buffer>p :call VOput()<cr>
```
Comment (mcepl) [inline vimoutliner/plugin/votl_smart_paste.vim]: What in the world is this?

File: vimoutliner/plugin/votl_smart_paste.vim [hunk 2]
```diff
@@ -80,17 +80,17 @@ function! VOput()
 	if (foldclosed(thisLine) == -1) && IsParent(thisLine)
 		normal! zc
 		let fold_cursor = getpos(".")
-		normal! ]p
```
Comment (mcepl) [inline vimoutliner/plugin/votl_smart_paste.vim]: OK, this looks good!

File: vimoutliner/plugin/votl_smart_paste.vim (hunk 1 above)
Comment (dahu) [inline vimoutliner/plugin/votl_smart_paste.vim]: What are you referring to?

Comment (mcepl) [inline vimoutliner/plugin/votl_smart_paste.vim]: To that ``Ctrl-U`` character which has been added.

**Comment by dahu** [comment]
:-)

That is not a typo. Look up   :help c_ctrl-u   . And ponder what the code
would do without that if the hapless user were to try something like: 3"*p

File: vimoutliner/plugin/votl_smart_paste.vim (hunk 1 above)
Comment (dahu) [inline vimoutliner/plugin/votl_smart_paste.vim]: That is not a typo.
Look up   `:help c_ctrl-u`
Ponder what the code would do without that if the hapless user were to try something like: `3"*p`

Comment (mcepl) [inline vimoutliner/plugin/votl_smart_paste.vim]: 1. How much it contributes to the fixing of the bug “Fix VOput() bug reported by grahameblackwood”?
2. It seems to me like you are second-guessing the current design of vim itself, not VO. I guess if you do that, you should get three times the content of "* register pasted. And so what?

Please, remove this chunk.

**Comment by dahu** [comment]
Correct. The intended result of 3"*p is to paste the content of the "*
register three times -- standard Vim behaviour. Standard Vim behaviour that
will not work in the votl_smart_paste.vim code without the <c-u> in the map
preceding the function call (to clear the erroneous range that the '3' in
the key sequence would generate).

You're babysitting the project, so it's your call. You can choose to have
broken behaviour or correct behaviour.
*I* won't remove it because I will not wilfully submit broken code.

**Comment by mcepl** [comment]
Thank you.

**Pull request merged**